packages/shared/src/components/comments/CommentBox.tsx

```diff
@@ -71,7 +71,7 @@ function CommentBox({
           user={comment.author}
           tooltip={{ appendTo: appendTooltipTo }}
         >
-          <ProfileImageLink user={comment.author} />
+          <ProfileImageLink className="z-1" user={comment.author} />
```

Comment:
This you could even always give it z-1 right? Instead of condtional bht up to you 👍

Reply:
Good point. I was just thinking it might affect different areas and wanted to keep positioning related properties to be given by the implementing component. Let's see what other think of this too 🚢